jdaviz/core/helpers.py

```diff
@@ -131,8 +131,17 @@ def plugins(self):
         plugins : dict
             dict of plugin objects
         """
-        return {item['label']: widget_serialization['from_json'](item['widget'], None).user_api
-                for item in self.app.state.tray_items}
+        plugins = {item['label']: widget_serialization['from_json'](item['widget'], None).user_api
+                   for item in self.app.state.tray_items}
+
+        # handle renamed plugins during deprecation
+        if 'Orientation' in plugins.keys():
+            plugins['Links Control'] = plugins['Orientation']._obj.user_api
+            plugins['Links Control']._deprecation_msg = 'in the future, the formerly named \"Links Control\" plugin will only be available by its new name: \"Orientation\".'  # noqa
+        if 'Canvas Rotation' in plugins.keys():
```

Comment:
Same as https://github.com/spacetelescope/jdaviz/pull/2179/files#r1457982295

```suggestion
        if 'Canvas Rotation' in plugins:
```